target/linux/ath79/dts/qca9533_tl-wr841.dtsi

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9533.dtsi"
+
+/ {
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x2000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-status = &system;
+	};
+
+	extosc: ref {
```

Comment:
What is this? Dead node? Isn´t referenced in your dts file...
Anyway the clock frequency is different from the one set in &ref { ... };

Reply:
maybe, dts was copied from 9567, I'll review it

Reply:
I looked over it and is related to wifi tries that I made